src/profile-logic/marker-timing.js

```diff
@@ -65,18 +65,18 @@ export function getMarkerTiming(markers: Marker[]): MarkerTimingRows {
   const markerTimingsMap: Map<string, MarkerTiming[]> = new Map();
 
   // Go through all of the markers.
-  for (let markerIndex = 0; markerIndex < markers.length; markerIndex++) {
-    const marker = markers[markerIndex];
+  for (let i = 0; i < markers.length; i++) {
+    const marker = markers[i];
```

Comment:
This is now just a loop index so I downgraded its variable name to something unnoticeable :)